packages/common-utils/src/__tests__/__snapshots__/renderChartConfig.test.ts.snap

```diff
@@ -64,7 +64,7 @@ exports[`renderChartConfig should generate sql for a single histogram metric 1`]
                   arrayElement(ExplicitBounds, BucketLowIdx - 1) + (arrayElement(ExplicitBounds, BucketLowIdx) - arrayElement(ExplicitBounds, BucketLowIdx - 1)) *
                     IF(arrayElement(CumRates, BucketLowIdx) > arrayElement(CumRates, BucketLowIdx - 1),
                       (Rank - arrayElement(CumRates, BucketLowIdx - 1)) / (arrayElement(CumRates, BucketLowIdx) - arrayElement(CumRates, BucketLowIdx - 1)), 0),
-                  IF(arrayElement(CumRates, 1) > 0, arrayElement(ExplicitBounds, BucketLowIdx + 1) * (Rank / arrayElement(CumRates, BucketLowIdx)), 0)
+                  arrayElement(ExplicitBounds, BucketLowIdx)
```

Comment:
Trying to understand this change. Does this mean we take the bound bucket if the value exceeds the boundaries? Also wonder what would happen in the old codes?